update_data.R

```diff
@@ -0,0 +1,313 @@
+# Script to update data ---------------------------------------
+setwd("C:/Users/lclem/OneDrive/Documents/GitHub/ParentAppDataScripts")
+
+update_data <- function(old_data_file = "plhdata_org_clean_dat.RDS"){
+  old_data <- readRDS(old_data_file)
+  var_names <- names(old_data)
+  
+  # rp.contact.field.parent_point_count_money_w_consequence, 
+  # parent_point_count_money_w_crisis
+  # rp.contact.field.parent_point_count_money_w_celebrate
+  #"rp.contact.field.parent_point_count_consequence_w_rules", "rp.contact.field.parent_point_count_consequence_w_crisis",
+  # dont exist. 
```

Comment:
@EsmeeIDEMS in addition, these variables commented above do not appear in the data frame. Do you happen to know why? Thanks.

Reply:
These fields only get created when a particular interaction happens. Simplest explanation is that no user has ever done that interaction. 

For example (see video below) `parent_point_count_money_w_crisis` = click on the `money` parent point whilst being in the workshop week `w_crisis`

https://user-images.githubusercontent.com/74557272/164717197-de9d3527-1919-461a-9cc4-80eb0137509d.mp4